orion/primitives/jsons/orion.primitives.tadgan.score_anomalies.json

```diff
@@ -1,76 +1,87 @@
-              "type": "ndarray"
-          }
-      ]
-  },
-  "hyperparameters": {
-      "tunable": {
-          "score_window": {
-              "type": "int",
-              "default": 10,
-              "range": [2, 200]
-          },
-          "critic_smooth_window": {
-              "type": "int",
-              "default": 200,
-              "range": [2, 400]
-          },
-          "error_smooth_window": {
-              "type": "int",
-              "default": 200,
-              "range": [2, 400]
-          },
-          "rec_error_type": {
-              "type": "str",
-              "default": "point",
-              "options": ["point", "area", "dtw"]
-          }
-      }
-  }
+    "name": "orion.primitives.tadgan.score_anomalies",
+    "contributors": [
+        "Alexander Geiger <geigera@mit.edu>",
+        "Dongyu Liu <dongyu@mit.edu>",
+        "Sarah Alnegheimish <smish@mit.edu>"
+    ],
+    "description": "Compute an array of anomaly scores based on a combination of reconstruction error and critic output",
+    "classifiers": {
+        "type": "postprocessor",
+        "subtype": "feature_extractor"
+    },
+    "modalities": ["timeseries"],
+    "primitive": "orion.primitives.tadgan.score_anomalies",
+    "produce": {
+        "args": [
+            {
+                "name": "y",
+                "type": "ndarray"
+            },
+            {
+                "name": "y_hat",
+                "type": "ndarray"
+            },
+            {
+                "name": "critic",
+                "type": "ndarray"
+            },
+            {
+                "name": "index",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "errors",
+                "type": "ndarray"
+            },
+            {
+                "name": "true_index",
+                "type": "ndarray"
+            },
+            {
+                "name": "true",
+                "type": "ndarray"
+            },
+            {
+                "name": "predictions",
+                "type": "ndarray"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "tunable": {
+            "score_window": {
+                "type": "int",
+                "default": 10,
+                "range": [2, 200]
+            },
+            "critic_smooth_window": {
+                "type": "int",
+                "default": 200,
+                "range": [2, 400]
+            },
+            "error_smooth_window": {
+                "type": "int",
+                "default": 200,
+                "range": [2, 400]
+            },
+            "rec_error_type": {
+                "type": "str",
+                "default": "point",
+                "options": ["point", "area", "dtw"]
+            },
+            "comb": {
+                "type": "str",
+                "default": "mult",
+                "options": ["mult", "sum", "res"]
+            },
+            "lam": {
```

Comment:
maybe use rec_lam or some words to indicate which type of error this lam is associated with?

Reply:
I think `lambda_rec` makes sense; to stay consistent with the definition of reconstruction, I'll change `res_scores` to `rec_scores`.